app/tree/gui/src/ts/twxplore/gui/geo/components/PanelParent/PanelParent.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from "react";
+import { Theme, createStyles, makeStyles } from '@material-ui/core/styles';
+import IconButton from '@material-ui/core/IconButton';
+import {TreesList} from "twxplore/gui/geo/components/trees/TreesList";
+import BoroughList from "../BoroughList/BoroughList";
+import CityList from "../CityList/CityList";
+import SpeciesList from "../SpeciesList/SpeciesList";
+import ArrowBackIcon from '@material-ui/icons/ArrowBack';
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    arrow: {
+        right: '0px',
+        marginBottom: '18px',
+        cursor: 'pointer',
+        '&:hover': {
+            color: 'blue'
+    }
+  },
+    arrow_disabled: {
+        color: 'grey'
+    }
+}))
+
+
+
+export const PanelParent: React.FunctionComponent<{}> = () => {
+    const classes = useStyles();
+
+    const callSetMode = (newMode) => {
+        setMode(newMode)
+    }
+    const [mode, setMode] = React.useState('home')
```

Comment:
Should be typed, `useState<string>` ?

Reply:
Mode should be an enum, in any case.

Reply:
If mode is an enum, how would my if-statements in my return work? 
That is, how can I use an enum to my advantage towards rendering the right components?